Runtime/Scripts/Export/GltfWriter.cs

```diff
@@ -153,9 +157,14 @@ struct AttributeData {
             m_Nodes.Add(node);
             return (uint) m_Nodes.Count - 1;
         }
-        
+
+        public void AddMeshAndSkinToNode(int nodeId, UnityEngine.Mesh uMesh, int[] materialIds)
+        {
+            throw new NotImplementedException();

```

Comment:
I'd expect a functioning, skin-less export in that case.
As a follow-up problem the method name would not reflect what it's doing.

Reply:
Maybe invoke the actual `AddMeshAndSkinToNode` below with `bones=null`.